waspc/src/Wasp/Generator.hs

```diff
@@ -77,3 +75,9 @@ writeDotWaspInfo dstDir = do
   let content = "Generated on " ++ show currentTime ++ " by waspc version " ++ show version ++ " ."
   let dstPath = dstDir </> [relfile|.waspinfo|]
   Data.Text.IO.writeFile (SP.toFilePath dstPath) (Data.Text.pack content)
+
+postGenDbGeneratorActions :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO ([GeneratorWarning], [GeneratorError])
```

Comment:
Not sold on the name here, but wanted a place to put DbGenerator things after gen, where one is a check that can produce warnings, and the other is an op that can produce errors.

Reply:
Sounds good, I will update the name and add a comment 👍🏻